### PR TITLE
[Cypress] Graph page find form test

### PR DESCRIPTION
**Pull request by matejnesuta**
Just a little suit for my PR involving logging in, visiting the graph page, typing something random into the find form and then expecting an error message to pop up.

Far from being perfect, but it is what it is.

(accidentally pushed some junk last time, now it should be alright)

### CODE REVIEW

**Comment by jmazzitelli** [comment]
Also, please provide a better title for this PR.. "Hopefully a better one" explains nothing about the PR.

**Comment by matejnesuta** [comment]
I was assigned to do my first PR as an intern in RH, so this is it. I apologize for the inconvenience and will make sure to do it properly the next time.

**Review by ScriptingShrimp — CHANGES_REQUESTED** [review]
Please address requested changes above.

File: frontend/cypress/integration/featureFiles/mnesuta_pr.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Feature: Kiali Graph page - Find/Hide 
+
+  Just a little suit for my PR involving logging in, visiting the graph page,
+  typing something random into the find form and then expecting an error message to pop up.
+
+  Background:
+    Given user is at administrator perspective
+    And user graphs "bookinfo" namespaces
```
[the remark this answers is not in the document]
Comment (matejnesuta) [reply frontend/cypress/integration/featureFiles/mnesuta_pr.feature:8]: https://github.com/kiali/kiali/issues/5329

Due to this, it is not possible to do a reasonable smoke test atm.

[the remark this answers is not in the document]
Comment (ScriptingShrimp) [reply frontend/cypress/integration/featureFiles/mnesuta_pr.feature:8]: @jmazzitelli can you please verify https://github.com/kiali/kiali/issues/5329 ?

File: frontend/cypress/integration/common/mnesuta_pr.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import {Then, When } from "cypress-cucumber-preprocessor/steps";
+
+When("user fills {string} in find and submits", (input: string) => {
+  cy.get('#graph_find').type(input+'{enter}')
+})
+
+Then("user sees the {string} message",(error: string)=> {
+  cy.get('[aria-label="graph settings"]').should("contain.text", error)
+})
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/common/mnesuta_pr.ts:9]: please add newline

**Review by ScriptingShrimp — DISMISSED** [review]
LGTM

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
The names of these files should change. "mnesuta_pr" ?? We don't name files with user names or with "pr"... the names of the files should briefly describe the test (its the title of the test, not the user who created it... and obviously its created via a PR so no need to put `_pr` in the name).

**Comment by jmazzitelli** [comment]
Is this all this test is going to do? Test that find/hide produces an error message when there is no graph nodes? Or is more going to be added to this? If this is all, then we should just add this little test in an existing cypress test rather than create another feature test suite. If more will be added later, then we can keep this test called "graph_smoke" and just add more smoke-testing to it. But I just want to make sure we aren't really committing to master a feature test suite with this little test as the only thing it will do.

**Comment by matejnesuta** [comment]
Well, I was assigned to do one simple test for my first PR, so I came up with this. Unfortunately, I am not able to answer your question and @ScriptingShrimp is offline for the rest of the day, but I hope he'll be able to answer you tomorrow.

**Comment by jshaughn** [comment]
There is already a `graph_find_hide.feature` file.  If you are going to add a test in this area I would suggest incorporating it into the existing test feature.

**Review by ScriptingShrimp — APPROVED** [review]
LGTM, rerunning pipeline after rebase

**Pull request merged**